yarn.lock

```diff
@@ -12266,6 +12304,14 @@ redoc@2.0.0-rc.29:
     tslib "^1.11.1"
     url-template "^2.0.8"
 
+redux@^4.0.1:
```

Comment:
Is it weird that it added redux, is that a dependancy of the addon?

Reply:
Redux is a dependency of Storybook. There must have been an update for some reason.